src/ctap/pin_protocol_v1.rs

```diff
@@ -345,10 +352,7 @@ impl PinProtocolV1 {
             item.copy_from_slice(&self.pin_uv_auth_token[i * 16..(i + 1) * 16]);
         }
         cbc_encrypt(&token_encryption_key, iv, &mut blocks);
-        let mut pin_token = vec![];
-        for item in blocks.iter().take(PIN_TOKEN_LENGTH / 16) {
-            pin_token.extend(item);
-        }
+        let pin_token: Vec<u8> = blocks.iter().flatten().cloned().collect();
```

Comment:
Nit: does `blocks.into_iter()` allow to remove the `.cloned()` later in the chain? Or is there no `into_iter` for arrays?